api/src/db/migrations/v143_add_raffle_bingo_games.ts

```diff
@@ -0,0 +1,804 @@
+import { v142Business, v142UserData } from "@db/migrations/v142_add_domestic_employer_roadmap_section";
+import { randomInt } from "@shared/intHelpers";
+
+export const migrate_v142_to_v143 = (v142Data: v142UserData): v143UserData => {
+  return {
+    ...v142Data,
+    businesses: Object.fromEntries(
+      Object.values(v142Data.businesses)
+        .map((business: v142Business) => migrate_v142Business_to_v143Business(business))
+        .map((currBusiness: v143Business) => [currBusiness.id, currBusiness])
+    ),
+    version: 143,
+  } as v143UserData;
+};
+
+export const migrate_v142Business_to_v143Business = (business: v142Business): v143Business => {
+  // @ts-expect-error this field exists for user data migrated from earlier than v108
+  delete business.formationData.businessNameSearch;
+
+  return {
+    ...business,
+    profileData: {
+      ...business.profileData,
+      raffleBingoGames: undefined,
```

Comment:
This is fine as a formal indicator that we're adding this to `profileData` object, but because there's no data actually being set this doesn't do anything.